### ISSUE TITLE
Watch target fails to detect changes

### ISSUE THREAD

**Issue by panesofglass** [panesofglass/reload#1]

**Comment by panesofglass** [comment]
Looks like `Watch` needs to call a function: https://github.com/fsharp/FAKE/issues/791

**Comment by panesofglass** [comment]
@forki, do you happen to know how I can fix the [Watch target](https://github.com/panesofglass/reload/blob/master/build.fsx#L31) so that it actually runs whenever a file is changed? Did I set something up incorrectly, by chance? My guess is that `System.Console.ReadLine` is blocking, but I'm probably incorrect.

**Comment by forki** [comment]
cc @tpetricek @voiceofwisdom

**Comment by panesofglass** [comment]
I was able to get this working following the guidance here: https://github.com/fsharp/FAKE/issues/791. I was also able to see the updates run when I ran in my terminal on OSX, where the files were hosted, but my Windows Console, hosted in Parallels and sharing the same folder, did not detect the file change. Looks like you'll need to have the files locally in Windows for the file change watcher to work correctly.
